src/appleseed.studio/python/module.cpp

```diff
@@ -68,7 +68,7 @@ Project* new_project()
 
 Project* open_project(const char* project_path)
```

Comment:
I don't know if it's a good idea. Maybe add function like `open_successfull` or make open_project return NULL (None) in case of error? I'm trying to imagine how script would look like and I don't like it very much
```Python
project = None
if studio.open_project(project_path):
    project = studio.current_project()
else:
    throw Exception(error_msg)
project.some_func()
```
what I don't like here is that semantic is wrong. You can expect that open_project returns opened project but it isn't. Also two different behaviors - performing action and querying information - are mixed here in `studio.open_project`

other two approaches:
1. open_successfull
```Python
project = studio.open_project(project_path)
if not studio.open_successfull():
    throw Exception(error_msg)
project.some_func()
```
2. return None
```Python
project = studio.open_project(project_path)
if not project:
    throw Exception(error_msg)
project.some_func()
```

Last one feels like "Python way" for me